department-of-veterans-affairs/caseflow
# Div/appeals 63282

Resolves [Instructional Text Update on Linked Appeals Gray Table](https://jira.devops.va.gov/browse/APPEALS-63282)

# Description
Made changes for the following instructional text is no longer visible in the linked appeal gray table
**'The linked appeal must be saved before tasks can be added.'**

## Acceptance Criteria

## Testing Plan


1. Sign in as INBOUND_OPS_TEAM_ADMIN_USER
2. Navigate to Queue
3. Click switch views and select Correspondences Cases from the dropdown.
4. Navigate to the completed tab to find a correspondence. Click any correspondence in the tab
5. On the Correspondence Details page, under Correspondence and Appeals Tasks tab
6. If there are no appeals that are linked under Existing appeals, try to link appeals by selecting the checkboxes under Existing appeals.
7. Then you will notice instructional text under linked appeal gray table The linked appeal must be saved before tasks can be added.
8. Click the save changes button.
9. Make sure to wait for a while until it is updated.
10. Verify the instructional text The linked appeal must be saved before tasks can be added. It should not be visible.


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
